returnn/tf/layers/basic.py

```diff
@@ -551,14 +551,16 @@ def __init__(self, activation, **kwargs):
       self.output.placeholder = self.output_before_activation.y
 
   @classmethod
-  def get_out_data_from_opts(cls, activation, **kwargs):
+  def get_out_data_from_opts(cls, sources, name, activation, **kwargs):
     """
+    :param list[LayerBase] sources:
+    :param str name:
```

Comment:
Sorry for the back and forth, but with my new suggestion, this is not needed anymore.

```suggestion
  def get_out_data_from_opts(cls, activation, **kwargs):
    """
```